src/sedcli_main.c

```diff
@@ -101,6 +103,12 @@ static cli_option write_mbr_opts[] = {
 	{0}
 };
 
+static cli_option blocksid_opts[] = {
+	{'d', "device", "Device node e.g. /dev/nvme0n1", 1, "DEVICE", CLI_OPTION_REQUIRED},
+	{'r', "hwreset", "Clear events by setting hardware Reset flag. Allowed values: 1/0", 1, "FMT", CLI_OPTION_REQUIRED},
```

Comment:
Spec has "hardware Reset" both capitilized. please stay consitent.

Reply:
I will change hardware to Hardware for consistency. As to the optional flag, I personally like to be more expilict so user knows exactly what they supply when it comes to issue BlockSID.